README.md

```diff
@@ -2,11 +2,72 @@
+- Red Hat ACM
+- [Baremetal Installer Provisioned Infrastructure] (https://docs.openshift.com/container-platform/4.12/installing/installing_bare_metal_ipi/ipi-install-overview.html)
+- [Crucible Ansible Playbooks] (https://github.com/redhat-partner-solutions/crucible)
+- [Agent Based Installer] (https://docs.openshift.com/container-platform/4.12/installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.html)
+
+Storage can be provided however you deem fit.  Cloud environments typically have dynamic storage available after OpenShift cluster installation.  The requirements are (2) PersistentVolumes available for consumption by the assisted-service and postgres containers in the assisted-service pod.  A few options exist that we can recommend:
+- OpenShift Data Foundation (ODF) is an option for providing dynamic storage provisioning for the resulting OpenShift cluster.
+- Local Storage Operator (LSO) is also an option in resource (CPU, memory, etc) constrained environments as an alternative to ODF.
+
 ## How to run it
+If this repository is private, you'll need to ensure that either your `token` or your `SSH key` is being used. You also need to create a secret for the repo in the `openshift-gitops` project, which is used to access the private GitHub repository.  It is suggested to use a team based SSH key, that way this repository could be used by multiple teams at Intel.
+
+### For HTTPS
+
+#### Generate token
+As the repo is private and you can not use SSH, you will have to generate an access token see [github's docs](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) for gudiance
+
 ```shell
-until oc apply -k https://github.com/redhat-partner-solutions/vse-carslab-hub/bootstrap/overlays/default; do sleep 3; done
+git clone https://<username>:<token>@github.com/redhat-partner-solutions/vse-cars-hub.git
 ```
 
+### For SSH
+
+#### Generate SSH Key 
+```shell
+ssh-keygen -q -C <key_name> -t ed25519 -f </path/to/key>
+```
+
+#### Upload the public key  
+Point your browser to [Intel-RedHat Deploy keys](https://github.com/redhat-partner-solutions/Intel-RedHat/settings/keys/new) to upload the contents of <key_name>. 
+
+<font size="1">NOTE: Please refer to [github's docs](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/managing-deploy-keys) for further information on managing deploy keys.</font>
+
+```shell
+git clone git@github.com:redhat-partner-solutions/vse-carslab-hub.git
+```
+
+#### Patch Storage Class
+```shell
+export KUBECONFIG=</path/to/kubeconfig>
+oc patch storageclass lso-filesystemclass -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"true"}}}'
+```
+
+#### Bootstrap
+```shell
+until oc apply -k vse-carslab-hub/bootstrap/overlays/default; do sleep 3; done
```

Comment:
I think it should be in the `How to run it` section up, as this is the first thing users see when reading the readme